platform/feature-api/api/src/pod-api/uri.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * Creates a random UUID string with a random hexadecimal value for each character in the string
+ * 'xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx', and returns the result.
+ * @returns a string in UUID format
+ */
+export function createUUID(): string {
+    return "xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx".replace(
```

Comment:
is there a reason we write this func ourselves here ? 🤔 

otherwise, we could use straight away the `uuid` lib
```import { v4 as uuid } from 'uuid';```
and install it by
```npm install --save-dev @types/uuid```
without reinventing the wheel...

Reply:
I just moved it from `browserPod`. It's always better, however, to not add dependencies, so I'm with whoever did this instead of adding a dep (I think it was @chisandrei, but I'd have to check)